flocker/node/backends.py

```diff
@@ -159,3 +159,13 @@ class BackendDescription(PClass):
 LOOPBACK = backend_loader.get('loopback')
 ZFS = backend_loader.get('zfs')
 GCE = backend_loader.get('gce')
+
+
+def backend_and_api_args_from_configuration(configuration):
+    """
+    """
```

Comment:
I hate this function, but I'll leave it and documented it for now. 
I think we can probably ditch it if / when we make our default backends into plugins. eg
``backend: flocker.node.agents.cinder``